### PR TITLE
Main Page Customization

### PR DESCRIPTION
**Pull request by marinasemen0va**

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -45,20 +45,5 @@
     {% else %}
     {{ content }}
     {% endif %}
-    <footer class="main-footer">
```
Comment (0ffz) [inline _layouts/home.html:48]: Keep the different footer in here

File: _sass/layout.scss [hunk 2]
```diff
@@ -19,15 +19,15 @@
   }
 
   @include mq(lg) {
-    width: calc((100% - #{$nav-width + $content-width}) / 2 + #{$nav-width});
+    width: calc((100% - #{$nav-width + $content-width}) / 2 + (#{$nav-width} - 50px));
```
Comment (0ffz) [inline _sass/layout.scss]: Revert for now

File: _sass/support/_variables.scss [hunk 3]
```diff
@@ -151,3 +151,5 @@ $media-queries: (
   lg: $content-width + $nav-width,
   xl: 1400px,
 ) !default;
+
+$header-image: url("../../assets/images/header.png") no-repeat 50% 50%;
```
Comment (0ffz) [inline _sass/support/_variables.scss]: Try to get this into html so we can use a config option & liquid

File: _sass/layout.scss [hunk 4]
```diff
@@ -19,15 +19,15 @@
...
     min-width: $nav-width;
   }
 }
 
 .main {
   @include mq(md) {
     position: relative;
-    max-width: $content-width;
+    max-width: $content-width + 50px;
```
Comment (0ffz) [inline _sass/layout.scss]: Revert this as well for now

**Review by 0ffz — CHANGES_REQUESTED** [review]
header.png should probably be in site later

File: _layouts/home.html (hunk 1 above)
Comment (marinasemen0va) [reply _layouts/home.html:48]: Moved footer.

File: _sass/layout.scss (hunk 2 above)
Comment (marinasemen0va) [reply _sass/layout.scss]: Done.

File: _sass/support/_variables.scss (hunk 3 above)
Comment (marinasemen0va) [reply _sass/support/_variables.scss]: Done.

**Pull request merged**